com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Settings/RenderPipelineSettingsUtilities.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace UnityEditor.Rendering.HighDefinition
+{
+    static class RenderPipelineSettingsUtilities
+    {
+        public static IEnumerable<string> Case1358409Workaround(IEnumerable<string> keywords)
```

Comment:
Do we really need a name like this? I guess there is no other way to do that right? So it's not really necessary to reference a bug case IMO.
So having something explicit like "RemoveDLSSKeywordIfNeeded" would be fine by me.

Reply:
As discussed offline, I'll wait for Julien to decide the naming